### PR TITLE
Added compliance testing page and references

### PR DESCRIPTION
**Pull request by HG-TWO**
- Compliance Testing page is added
- Added reference in IG Information
- Added reference in introduction

### CODE REVIEW

File: docs/api-publishing/02-Components.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ API consumer application developers will expect to be able to follow a simple an
 
 API Providers **MUST** provide a developer onboarding function that enables API Consumer developers to create and manage their client application credentials and request application level access to API Provider APIs.
 
+<ApiStandard id="HNZAS_MUST_have_compliance" type="MUST">API providers **MUST** provide compliance tests that API consumers are required to pass in order to onboard to In Confidence or higher-level APIs.</ApiStandard>
```
Comment (swithinfoote) [inline docs/api-publishing/02-Components.md:55]: Should this perhaps be slightly re-worded to say something along the lines of `MUST provide minimum compliance requirements to API Consumers` and `artefacts SHOULD be available via a web experience`?

File: docs/api-publishing/08-ComplianceTesting.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+- Provide test report or summary template  
+
+- Consider alternative or innovative tools to ease the compliance testing process, e.g. recorded demo, automation, report generation, etc. 
+
+## Compliance testing process 
+
+*API Provider* **MUST** provide compliant testing process definition. Diagrams **SHOULD** be used that describe an API Consumer compliance testing journey. 
+ 
+The process of compliance testing involves several steps to verify that the system meets the required compliance. Below is how a typical compliance test process looks like:
```
Comment (ChrisSquats) [inline docs/api-publishing/08-ComplianceTesting.md:65]: Can you update the diagram to be clear on who is doing what? i.e. API consumer tasks vs provider tasks.

File: docs/api-publishing/08-ComplianceTesting.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    - Iterate until compliance is achieved: Repeat the process of fixing and testing until all compliance requirements are met. 
+
+
+1. Ongoing Compliance Monitoring (if necessary)
+    API Provider can require continuous monitoring or periodic reassessment to ensure ongoing compliance. When compliance requirements are changed, ensure that the API Consumer undergoes updated compliance testing.
+
+## Examples 
+
+- Get Match And Validate Patient Compliance Testing
+    https://master.d3b08qop7whnnl.amplifyapp.com/GetMatchAndValidatePatientComplianceTesting.html
```
Comment (ChrisSquats) [inline docs/api-publishing/08-ComplianceTesting.md:132]:
```suggestion
https://nhi-ig.hip.digital.health.nz/GetMatchAndValidatePatientComplianceTesting.html
```

**Review by ChrisSquats — CHANGES_REQUESTED** [review]

File: docs/api-publishing/08-ComplianceTesting.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+**Improved quality**: Enhances the overall services quality, security, and performance; improves data quality and data integrity by embedding the quality requirements into design and assurance process.
+
+## Key Considerations of Compliance Testing
+API Provide **SHOULD** take the following key considerations during the design of the compliance test:
+
+- Align Compliance Testing to business use cases, ensuring the process is easy to adopt 
+
+- Provide clear instructions and criteria (e.g. using [Gherkin syntax](https://cucumber.io/docs/gherkin/reference/)) 
+
+- Provide information of mandatory tests and conditions (e.g. a test is only required to complete if API Consumer has permission to view a patient's contact details)
```
[the remark this answers is not in the document]
Comment (HG-TWO) [reply docs/api-publishing/08-ComplianceTesting.md:30]: Not sure about this one. This covers the conditional requirements of compliance testing. Here is an example: 
image

File: docs/api-publishing/02-Components.md (hunk 1 above)
Comment (HG-TWO) [reply docs/api-publishing/02-Components.md]: Included in the PR. Pls review.

File: docs/api-publishing/08-ComplianceTesting.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+## Purpose 
+
+The goal of compliance testing is to ensure that the API consumer application meets the compliance requirements, technical specifications, and legal regulations imposed by API Provider. 
+
+### Key Benefits of Compliance Testing: 
+
+**Legal and regulatory adherence**: Ensures that the product or system complies with laws and regulations to avoid legal penalties. 
+
+**API Provider and API Consumer confidence**: Demonstrates that API consumption meets compliance requirements, which can help build confidence with API Providers, API Consumers and stakeholders.
```
[the remark this answers is not in the document]
Comment (HG-TWO) [reply docs/api-publishing/08-ComplianceTesting.md:15]: Made changes. Pls review.

File: docs/api-publishing/08-ComplianceTesting.md (hunk 2 above)
Comment (HG-TWO) [reply docs/api-publishing/08-ComplianceTesting.md:98]: Added actors. Pls review.

File: docs/api-publishing/08-ComplianceTesting.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ChrisSquats) [reply docs/api-publishing/08-ComplianceTesting.md:30]: agree

File: docs/api-publishing/08-ComplianceTesting.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+
+- Provide test report or summary template  
+
+- Consider alternative or innovative tools to ease the compliance testing process, e.g. recorded demo, automation, report generation, etc. 
+
+## Compliance testing process 
+
+API Provider **MUST** provide compliant testing process definition. Diagrams **SHOULD** be used that describe an API Consumer compliance testing journey. 
+ 
+The process of compliance testing involves several steps to verify that the system meets the required compliance. Below is how a typical compliance test process looks like:
```
Comment (ChrisSquats) [inline docs/api-publishing/08-ComplianceTesting.md:98]:
```suggestion
The process of compliance testing involves several steps to verify that the system meets the required standards. Below is an overview of a typical compliance testing process:
```
minor wording fix

File: docs/api-publishing/08-ComplianceTesting.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+- Provide test report or summary template  
+
+- Consider alternative or innovative tools to ease the compliance testing process, e.g. recorded demo, automation, report generation, etc. 
+
+## Compliance testing process 
+
+API Provider **MUST** provide compliant testing process definition. Diagrams **SHOULD** be used that describe an API Consumer compliance testing journey. 
+ 
+The process of compliance testing involves several steps to verify that the system meets the required standards. Below is an overview of a typical compliance testing process:
+```plantuml alt="Example of Compliance Testing Process"
```
Comment (ChrisSquats) [inline docs/api-publishing/08-ComplianceTesting.md:99]:
```suggestion
```plantuml alt="Flowchart illustrating the compliance testing process between an API Provider and API Consumer. It begins with the API Consumer identifying test requirements and guidelines, followed by a request for compliance testing. Test planning occurs with the API Provider preparing the test environment, while the API Consumer prepares the tests. The API Consumer then executes the tests and records the outcomes. Results are evaluated, and if necessary, remediation is performed. The cycle repeats until compliance is certified. Once certified, compliance monitoring is conducted by the API Provider and continues while ongoing."
```

Comment (HG-TWO) [reply docs/api-publishing/08-ComplianceTesting.md:99]: Should this be merged into the elaboration below the diagram?

File: docs/api-publishing/08-ComplianceTesting.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+1. Remediation (if necessary)
+    - Fix the identified non-compliance issues 
+    - Re-test after corrections: After fixes have been implemented, re-run the necessary tests to ensure compliance. 
+
+    - Iterate until compliance is achieved: Repeat the process of fixing and testing until all compliance requirements are met. 
+
+
+1. Ongoing Compliance Monitoring (if necessary)
+    API Provider can require continuous monitoring or periodic reassessment to ensure ongoing compliance. When compliance requirements are changed, ensure that the API Consumer undergoes updated compliance testing.
+
```
Comment (ChrisSquats) [inline docs/api-publishing/08-ComplianceTesting.md]: Suggest we remove this. Too much detail for a standard.

**Review by ChrisSquats — APPROVED** [review]

**Review by swithinfoote — APPROVED** [review]
Looks good, thanks for making the changes

**Pull request merged**